pkg/vault/test/authorizer/TimelockAuthorizerPermissions.test.ts

```diff
@@ -665,6 +598,48 @@ describe('TimelockAuthorizer permissions', () => {
             });
           });
         });
+
+        context('when the user has the permission in a specific contract and everywhere', () => {
+          sharedBeforeEach('grants the permissions', async () => {
+            await authorizer.grantPermission(ACTION_1, user, WHERE_1, { from: root });
+            await authorizer.grantPermissionGlobally(ACTION_1, user, { from: root });
+          });
+
+          context('when revoking the permission for a contract', () => {
+            it('cannot revoke the permission', async () => {
+              await expect(
+                authorizer.revokePermission(ACTION_1, user, WHERE_1, { from: getSender() })
+              ).to.be.revertedWith('ACCOUNT_HAS_GLOBAL_PERMISSION');
+            });
+          });
+
+          context('when revoking the permission for a everywhere', () => {
+            it('revokes the requested global permission', async () => {
+              await authorizer.revokePermissionGlobally(ACTION_1, user, { from: getSender() });
+
+              expect(await authorizer.canPerform(ACTION_1, user, EVERYWHERE)).to.be.false;
+              expect(await authorizer.canPerform(ACTION_1, user, WHERE_2)).to.be.false;
+            });
+
+            it('can still perform the requested action in the specific contract', async () => {
+              await authorizer.revokePermissionGlobally(ACTION_1, user, { from: getSender() });
+
+              expect(await authorizer.canPerform(ACTION_1, user, WHERE_1)).to.be.true;
```

Comment:
Could also check that they can't do WHERE_2:
              expect(await authorizer.canPerform(ACTION_1, user, WHERE_2)).to.be.false;

Reply:
Note that this is already done in the test above.